webpack.config.js

```diff
@@ -0,0 +1,19 @@
+const config = {
+    mode: 'none', // "production" | "development" | "none"
+    resolve: {
+        extensions: ['*', '.mjs', '.js', '.json']
```

Comment:
I am not super familiar with this file but is there a reason we would include a `*` (wildcard) here for file extensions?

Reply:
I'm not sure either, but this code was suggested to resolve package import errors.